docs/articles/user_acquisition_analytics.md

```diff
@@ -0,0 +1,442 @@
+
+In this article, we'll show you how to use the Superlinked library to overcome these challenges - letting you leverage vector embedding power to **identify and analyze users on the basis of how they respond to particular ad messaging, target them more precisely, and improve conversion rates in your campaigns**.
+
+## Vector embedding - power & challenges
+
+By capturing intricate relationships and patterns between data points and representing them as high-dimensional vectors in a latent space, embeddings empower you to extract deeper insights from complex datasets, thereby enabling more nuanced analysis, interpretation, and accurate predictions to inform your ad campaign decision-making.
+
+But while vector embeddings are a powerful tool for user analysis, they also introduce **additional challenges**:
+
+- *Quality and relevance* - to achieve good retrieval results and avoid postprocessing and reranking, embedding generation techniques and parameters need to be selected carefully
+- *Scalability with high-dimensional data* - rich data increases computational complexity and resource requirements, especially when working with large datasets
+- *Interpretability* - identifying underlying patterns and relationships (e.g., how specific users respond to certain ad types) embedded in abstract vectors can be tricky
+
+## Smarter vectors
+
+We can use Superlinked's framework to overcome these challenges by creating vectors that are smarter representations of your data and therefore let you retrieve high quality, actionable insights (e.g., understanding why different users respond to different ad creatives) without postprocessing or reranking.
+
+Let's walk through how you can perform user acquisition analytics on ad creatives using Superlinked library elements, namely:
+
+- **Recency space** - to understand the freshness of information
+- **Number space** - to interpret user activity
+- **TextSimilarity space** - to interpret the ad creatives
+- **Query time weights** - to optimize results by defining how you treat your data when you run a query, without needing to re-embed the whole dataset
+
+## User data
+
+We have data from two 2023 ad campaigns - one from August (with more generic ad messages), and another from December (assisted by a made-up influencer, "XYZCr$$d"). Our data (for 8000 users) includes:
+
+1. signup date, as unix timestamp
+2. the ad creative a user clicked on before signing up
+3. average (user) daily activity, measured in API calls/day (over the user's lifetime)
```

Comment:
@morkapronczay
Below (at line 170) you say our data are exclusively campaign-related. Are the API calls exclusively ad_creative clicks?